api/versions.py

```
@@ -0,0 +1,97 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+class RecordVersionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = RecordVersion
+        fields = ("version", "prev_version", "changed_at", "diff", "meta")
+
+
+class VersionViewSet(viewsets.ViewSet):
+    permission_classes = [IsAuthenticated]
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)